download-ipfs-binary.bash

```diff
@@ -0,0 +1,45 @@
+#!/bin/bash
```

Comment:
I am thinking if we should have this approach or download during the initial Loading (if the binary is not available).
The main reason is that the binary will be packed into the `.asar` package, and so it should not be able to spawn a process.

What do you think @0x6431346e ?

Reply:
Read more here: https://stackoverflow.com/questions/38361996/how-can-i-bundle-a-precompiled-binary-with-electron/43389268#43389268

This approach of including the binary in the package, is not suggested apparently.

Reply:
As discussed I am fine with this current solution, but we will update it if some requests arrives.